src/Article.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from './utils/api';
+
+function Article() {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState(null);
+
+  useEffect(() => {
+    getArticleById(article_id)
+      .then(data => {
+        setArticle(data.article);
+      })
+      .catch(error => {
+        console.error('Error fetching article:', error);
+      });
+  }, [article_id]);
+
+  if (!article) {
```

Comment:
With loading state, we could replace this logic. At the moment this would forever show the p tag saying it's loading, even if nothing is ever going to arrive!